sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -565,16 +680,31 @@ class depends_on {
   public:
     template<typename... NodeTN>
     depends_on(NodeTN... nodes);
+
+    template<typename... EventTN>
+    depends_on(EventTN... events);
 };
 }
 ----
 
 The API for explicitly adding nodes to a `command_graph` includes a
 `property_list` parameter. This extension defines the `depends_on` property to
-be passed here. `depends_on` defines any `node` objects for the created node to
-be dependent on, and therefore form an edge with. These nodes are in addition to
-the dependent nodes identified from the command-group requisites of the created
-node.
+be passed here. `depends_on` may be used in two ways:
+
+* Passing nodes from the same `command_graph` which will create dependencies and
+graph edges between those nodes and the node being added. 
+
+* Passing SYCL events, including <<dynamic-events, Dynamic Events>>. If an event
+is a <<event-terminology, graph-limited event>> from the same `command_graph`,
+then a graph edge is created between this node and the other node. For dynamic
+events, graph-limited events from a different graph or <<event-terminology,
```

Comment:
```suggestion
events or events, graph-limited events from a different graph or <<event-terminology,
```

If EventTN is event, two cases: 
1) graph-limited event of the same graph (described in the spec at line 698),    
2) graph-limited event of a different graph and regular sycl events (not described in the spec, and the above change is for this)